tests/phpunit/tests/interactivity-api/wpInteractivityAPI.php

```diff
@@ -639,7 +660,8 @@ public function test_process_directives_changes_html_if_contains_svgs() {
 			</header>
 		';
 		$processed_html = $this->interactivity->process_directives( $html );
-		$p              = new WP_HTML_Tag_Processor( $processed_html );
+		$this->setExpectedIncorrectUsage( 'WP_Interactivity_API::process_directives_args' );
```

Comment:
This is available as an annotation. Add this to the tests that are expected to fail and all the changes to implement this tracking can be removed:

```
	 * @expectedIncorrectUsage WP_Interactivity_API::process_directives_args
```